editor/components/default-block-appender/index.js

```diff
@@ -9,27 +9,16 @@ import 'element-closest';
  */
 import { __ } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
-import { getDefaultBlockName, createBlock } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import BlockDropZone from '../block-drop-zone';
-import { insertBlock } from '../../actions';
+import { appendDefaultBlock } from '../../actions';
 import { getBlockCount } from '../../selectors';
 
 export class DefaultBlockAppender extends Component {
```

Comment:
I guess this could be a functional component now :)

Reply:
My thinking too, but jest wasn't happy and I left it for now :)